### ISSUE TITLE
Write test for critical functions

### ISSUE THREAD

**Issue by JhohannesK** [JhohannesK/TipoHiZ#72]

**Comment by JhohannesK** [comment]
so we have critical functions which the apply really relies on to work well. You can find these functions under the utils folder.

To prevent other devs from breaking these functions, we need a test that make sure these functions are always correct before even commited.

**Comment by RonakSurana-2001** [comment]
Understood you want me to write unit test(eg. Jest) for the useHandleText.ts and other files in utils folder

**Comment by JhohannesK** [comment]
@RonakSurana-2001 Are you handling this?

**Comment by RonakSurana-2001** [comment]
I have to write jest test cases right asking

**Comment by JhohannesK** [comment]
let's use vitest.
Are you fine with it?

**Comment by RonakSurana-2001** [comment]
Yes I have never used vitest so it will be great learning for me too
![Screenshot_2024-10-06-08-24-59-516_com android chrome](https://github.com/user-attachments/assets/c594592a-60a0-4e16-80ff-984556bbe94d)
I have to write for these files

**Comment by JhohannesK** [comment]
Leave util.tsx out for now

**Comment by JhohannesK** [comment]
@RonakSurana-2001 join the discord let's continue the communication there
https://github.com/JhohannesK/TipoHiZ/issues/99
